bots: Move fedora-i386 image to Fedora 29

martinpitt opened the pull request:
* [x] image-refresh fedora-i386

cockpituous commented:
image-refresh fedora-i386 done: https://github.com/cockpituous/cockpit/commits/image-refresh-fedora-i386-20181108-154444

martinpitt commented:
Looks like abrt still doesn't work well enough on i386, so I may have to revert the test case enabling. But I'll have a closer look.

martinpitt commented:
Journal says

    abrt-dump-journal-core[428]: Failed to obtain all required information from journald

which is still as broken as in Fedora 28, so I brought back the skip and adjusted the message.

I also adjusted check-storage-resize.
